Source/JavaScriptCore/runtime/InitializeThreading.cpp

```diff
@@ -49,13 +49,17 @@
 #include <wtf/Threading.h>
 #include <wtf/threads/Signals.h>
 
+WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN
```

Comment:
Should we use the header-specific pragma?

Reply:
Did you mean WTF_IGNORE_WARNINGS_IN_THIRD_PARTY_CODE_BEGIN? I think WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN is slightly better here because in theory we could fix it.

Reply:
Yes, I meant `WTF_IGNORE_WARNINGS_IN_THIRD_PARTY_CODE_BEGIN`. I thought that's what we used around headers. Yes we can fix it, but not here. Someone would have to go fix those headers. No big deal either way. I just preferred if this didn't show in my usual searches for `WTF_ALLOW_UNSAFE_BUFFER_USAGE_BEGIN`

Reply:
Make sense. I can do that in a follow up.